pkg/limayaml/limayaml.go

```diff
@@ -243,6 +246,18 @@ type CACertificates struct {
 	Certs          []string `yaml:"certs,omitempty" json:"certs,omitempty"`
 }
 
+type GuestAgentProto = string
+
+const (
+	GuestAgentUNIXProto  GuestAgentProto = "UNIX"
+	GuestAgentVSockProto GuestAgentProto = "VSOCK"
+	GuestAgentTCPProto   GuestAgentProto = "TCP"
```

Comment:
These should be in lower characters we are going to support customizing the protocol, but probably we can just make this uncustomizable